tracks/typescript/exercises/resister-color-duo/mentoring.md

```diff
@@ -0,0 +1,45 @@
+### Reasonable Solutions
```

Comment:
Have a look the JavaScript mentor notes which has a more idiomatic approach and _more_ allowed variations.